### DISCUSSION TITLE
Unable to send invitation mail normally(smtp.exmail.qq.com)

### DISCUSSION THREAD

**Discussion by asasdsaweasd** [dani-garcia/vaultwarden#3309] [Q&A]
He can work in testing mail and verifying mailbox functions, but he cannot work in sending invitation mail. How should I solve it?

Version: docker:latest(1.27)
![image](https://user-images.githubusercontent.com/47250180/222952202-5e98c101-73f1-4b30-8d21-d29dc043317c.png)
![image](https://user-images.githubusercontent.com/47250180/222952577-a1fe38f1-073f-48a7-adbb-3257d682b85f.png)

![image](https://user-images.githubusercontent.com/47250180/222952281-11534577-fa6b-4b24-8944-44206da5f509.png)

{"ErrorModel":{"Message":"SMTP 5xx error: permanent error (550): Error: content rejected.http://mail.qq.com/zh_CN/help/content/rejectedmail.html","Object":"error"},"ExceptionMessage":null,"ExceptionStackTrace":null,"InnerExceptionMessage":null,"Message":"SMTP 5xx error: permanent error (550): Error: content rejected.http://mail.qq.com/zh_CN/help/content/rejectedmail.html","Object":"error","ValidationErrors":{"":["SMTP 5xx error: permanent error (550): Error: content rejected.http://mail.qq.com/zh_CN/help/content/rejectedmail.html"]},"error":"","error_description":""}

**Comment by BlackDex** [comment]
Your mail provider is blocking because of something which is in the body of the mail.
That is not something we can fix. For one, we don't know what part of the content is being blocked, and all other mail providers like Google, Microsoft, Yahoo etc.. seem to except these mails without marking them as spam or having content to be rejected.

Either try to talk with your mail provider if they can tell you why this mail is blocked, or use a different mail provider.
One solution could be using mailing providers like SendGrid or MailJet maybe. See https://github.com/dani-garcia/vaultwarden/wiki/SMTP-Configuration#smtp-servers

**Reply by asasdsaweasd** [reply]
Thank you for your reply
